examples/http_server.md

```diff
@@ -83,7 +83,7 @@ const handler = (request: Request): Response => {
 };
 
 console.log(`HTTP webserver running. Access it at: http://localhost:8080/`);
-await listenAndServe(addr, handler);
+await serve(handler, { addr } );
```

Comment:
Btw. Not sure if `await` is correct here. It isn’t used in the [docs](https://doc.deno.land/https/deno.land/std@0.114.0/http/mod.ts#serve).

Reply:
This is fine